jest.config.js

```diff
@@ -176,9 +176,10 @@ module.exports = {
   // transform: null,
 
   // An array of regexp pattern strings that are matched against all source file paths, matched files will skip transformation
-  // transformIgnorePatterns: [
-  //   "/node_modules/"
-  // ],
+  transformIgnorePatterns: [
+    "/node_modules/",
```

Comment:
You are the JS expert and lover but is it OK to ignore transpiling `node_modules`? (genuine question, not trolling you)

Reply:
this is just for jest, and tests are running in node so nothing really needs transpiling. You do want this transpiled in the "babel" step so `node_modules` aren't excluded in the other file (package.json -> babel.ignore)